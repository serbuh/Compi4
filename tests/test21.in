// Dead while statements
while(false) {
  print "Shouldn't be printed #1";
}

C x = 100 C;
C y = 200 F;

while(x == y) {
  print "Shouldn't be printed #2";
}

print "Only this line should be printed";
