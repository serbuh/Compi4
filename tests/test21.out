Only this line should be printed
